disco/pipelines/template/time-series-default-template.toml

```diff
@@ -1,3 +1,4 @@
+task_name = "DISCO Task"
```

Comment:
Would it be better to not have a default value?

Reply:
Removed the default value. It's required now, even there's no db ingestion, like for `upgrade`.